packages/entity-server/src/models/Project.ts

```diff
@@ -13,6 +13,7 @@ type ProjectFields = Readonly<{
   name: string;
   entity_id: string;
   entity_hierarchy_id: string;
+  permission_groups: string[];
 }>;
```

Comment:
Optional, but feel free to sub out this whole `ProjectFields` type with the `Project` type from `@tupaia/types`. We're steadily cleaning all these duplicate type defs up.